package.json

```diff
@@ -44,27 +44,18 @@
     },
     "devDependencies": {
         "@eslint/js": "^9.35.0",
-        "@types/chai": "^4.3.20",
         "@types/diff": "^5.2.3",
-        "@types/mocha": "^10.0.10",
         "@types/node": "^22.18.5",
-        "@types/sinon": "^10.0.20",
         "@types/vscode": "^1.90.0",
-        "@vscode/test-cli": "^0.0.11",
-        "@vscode/test-electron": "^2.5.2",
         "@vscode/vsce": "^3.6.0",
-        "chai": "^4.5.0",
         "esbuild": "^0.25.9",
         "eslint": "^9.35.0",
         "husky": "^9.1.7",
         "jest": "^30.1.3",
         "jest-mock-vscode": "^4.7.0",
-        "mocha": "^10.8.2",
-        "npm-run-all": "^4.1.5",
```

Comment:
Is [this](https://www.npmjs.com/package/run-s?activeTab=versions) where `run-s` came from?